internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -433,7 +433,7 @@ func LoadConfig(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFu
 
 				b, e = receiveConfigViaMaintenanceService(ctx, logger, r)
 				if e != nil {
-					return fmt.Errorf("failed to receive config via maintenance service: %w", err)
+					return fmt.Errorf("failed to receive config via maintenance service: %w", e)
```

Comment:
this bug was driving me crazy, as it shows totally wrong error message here :face_with_thermometer: